File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Ranking.java
package fr.jbme.raiderioapp.data.model.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Ranking {

    @SerializedName("world")
    @Expose
    public Integer world;
    @SerializedName("region")
    @Expose
    public Integer region;
    @SerializedName("realm")
    @Expose
    public Integer realm;

}

File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Ranking.kt
package fr.jbme.raiderioapp.data.model.character

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Ranking(
    @SerializedName("world")
    @Expose
    var world: Int? = null,

    @SerializedName("region")
    @Expose
    var region: Int? = null,

    @SerializedName("realm")
    @Expose
    var realm: Int? = null
)